loklak.php

```diff
@@ -186,4 +186,49 @@ public function suggest($query="", $count=null, $order=null, $orderby=null, $sin
 			return json_encode($request, true);
 		}
 	}
+
+	public function account($name, $action=null, $data=null) {
+		$this->requestURL = 'http://localhost:9000/api/settings.json';
+		$this->name = $name;
+		$this->action = $action;
+		$this->data = $data;
```

Comment:
Write a comment here that the `$data` to be sent to the function should be stringified `json`